include/SPTDataLoader/module.modulemap

```diff
@@ -0,0 +1,3 @@
+module SPTDataLoader {
+    umbrella "."
```

Comment:
Could we use the [`SPTDataLoaderConvenience.h`](https://github.com/spotify/SPTDataLoader/blob/master/include/SPTDataLoader/SPTDataLoaderConvenience.h) umbrella header instead?